src/robusta/core/playbooks/job_utils.py

```diff
@@ -5,12 +5,31 @@
 CONTROLLER_UID = "controller-uid"
 
 
+def get_job_selector(job: Job) -> Optional[str]:
+    """
+    First try to get by controller_uid, then by selectors, and if both not available by name
+    """
+    job_labels = job.metadata.labels
+    controller_uid = job_labels.get(CONTROLLER_UID, None)
+    if controller_uid:
+        return f"{CONTROLLER_UID}={controller_uid}"
+    elif job.spec.selector and job.spec.selector.matchLabels:
+        return ",".join(f"{k}={v}" for (k,v) in job.spec.selector.matchLabels.items())
```

Comment:
you can use build_selector_query(job.spec.selector)

Reply:
fixing